buildenv/jenkins/common/build.groovy

```diff
@@ -302,7 +302,14 @@ def archive_sdk() {
             def archiveCmd = SPEC.contains('zos') ? 'pax -wvz -p x' : 'tar -cvz -T -'
             // Filter out unwanted files (most of which are available in the debug-image).
             def filterCmd = "sed -e '/\\.dbg\$/d' -e '/\\.debuginfo\$/d' -e '/\\.diz\$/d' -e '/\\.dSYM\\//d' -e '/\\.map\$/d' -e '/\\.pdb\$/d'"
-            sh "( cd ${buildDir} && find ${JDK_FOLDER} -type f | ${filterCmd} | ${archiveCmd} ) > ${SDK_FILENAME}"
+            // Rename the top level directory for JDK11 z/OS to J11.0_64.
+            def ZOS_JDK_FOLDER = "J11.0_64"
```

Comment:
As I asked in the previous version of this PR, why is this being hardcoded here?  There's no rationale or description of why this is required.

Reply:
@DanHeidinga Agreed that the hard-coding shouldn't be there. Will look at a way to utilize the variables available and define "ZOS_JDK_FOLDER". 

As far as the folder name is concerned, this is pretty much z/OS Installable package (SMP/E) specific issue. SMP/E package is created from the pax.Z. Main reason being that z/OS supports multiple JDK installations on a system and hence, the name of the folder will differentiate the JDK versions (JDK7/8/11) installed on the same system.

Reply:
Thanks for the clarification.  If there's a better way to use ZOS_JDK_FOLDER or a similar variable to define the "root" directory for the zip (or I guess pax.Z), it would fit in this project.  A zOS-specific packaging change is a less clear that it belongs here and not somewhere closer to the zOS packaging system